addon/components/lmb-plugin/search-modal.ts

```diff
@@ -39,13 +39,24 @@ export default class LmbPluginSearchModalComponent extends Component<Args> {
   }
 
   fetchData = restartableTask(async () => {
```

Comment:
Not really related to this PR: I don't really understand why this is a `restartableTask`. The `this.error` field assignment also doesn't really seem necessary to me, as the `lastComplete`/`last` already has an `error` field.
It works, but ideally I think a `trackedFunction` might be more suited in this situation. Ideally, I also think that the `search` function shouldn't really be a `task` instance at all, as it doesn't really run any async operations (except for this single `fetchData` run), but not sure about that.

Reply:
I think it's a `restartableTask`  as changing the search term will re-run the search. We're not using an `AbortController` though, so I don't think it's adding anything right now over being a `trackedFunction`.

Reply:
Actually, just looked again and realised that the search is just filtering on the results, so indeed it should not be restartable.